### ISSUE TITLE
java.lang.UnsatisfiedLinkError: 'long com.luciad.imageio.webp.WebPDecoderOptions.createDecoderOptions() on M1 Macbook

### ISSUE THREAD

**Issue by joetsaitw** [usefulness/easylauncher-gradle-plugin#275]
Hey EasyLauncher!

I faced a WebP issue on M1 Macbook:
```
Caused by: java.lang.UnsatisfiedLinkError: 'long com.luciad.imageio.webp.WebPDecoderOptions.createDecoderOptions()'
	at com.luciad.imageio.webp.WebPDecoderOptions.createDecoderOptions(Native Method)
	at com.luciad.imageio.webp.WebPDecoderOptions.<init>(WebPDecoderOptions.java:26)
	at com.luciad.imageio.webp.WebPReadParam.<init>(WebPReadParam.java:24)
	at com.luciad.imageio.webp.WebPReader.getDefaultReadParam(WebPReader.java:147)
```

I've already tried v4.1.0, v4.2.0, v5.0.0, and none of them work.
Could you please check this issue? Thanks!

**Comment by joetsaitw** [comment]
> 

@mateuszkwiecinski Is there anything I can help with?

**Comment by mateuszkwiecinski** [comment]
If you can help then sure 😄 This plugin needs an `ImageIO` plugin that supports both reading and writing to WebP files. Something like https://github.com/sejda-pdf/webp-imageio/pull/6 or https://github.com/haraldk/TwelveMonkeys/issues/659 which supports all existing architectures + the problematic here M1. 
Which means someone has to publish an open-source library with appropriate licence which can replace the currently used ones

**Comment by viakunin** [comment]
I've opened the PR with the missing binary here - https://github.com/gotson/webp-imageio/pull/1

**Comment by mateuszkwiecinski** [comment]
Hello again 👋  
I just published a new version, that should contain the binary compiled by @viakunin. I haven't found a way to include ARM macs in CI pipeline or even test it manually, so I don't even know if it works 😬 
Please let me know if the most recent version does or does not work on your setup!

And just in case that's useful for anyone: the plugin will now use fork of a fork of a fork of a fork of a WebP decoding library: https://github.com/usefulness/webp-imageio - might be useful when debugging things 😉

**Comment by mateuszkwiecinski** [comment]
I was told this is fixed in 5.1.1 ✅
